.github/workflows/build.yml

```diff
@@ -81,20 +64,20 @@ jobs:
       - name: Cache dependencies
         uses: actions/cache@v2
         with:
-          path: ${{ matrix.cache_path }}
+          path: ~/.stack
           key: ${{ matrix.os }}-${{ hashFiles('stack.yaml') }}
 
       - name: Build & test
         id: build_test
         run: |
-          stack build ${{ matrix.stack_args }}
+          stack test
```

Comment:
I was doing `stack build --test` even when it wasn't parameterized.. let's keep that - it's more explicit that it also builds